doc/dev/generate.md

```diff
@@ -0,0 +1,96 @@
+<!-- cspell: ignore tspconfig mgmt -->
+
+# Generating Management Plane Crates from TypeSpec
+
+This document complements [Creating a TypeSpec-Based Rust SDK Client](https://github.com/Azure/azure-sdk-for-rust/blob/main/doc/dev/new-typespec-based-client.md). Read that guide first for the canonical, step-by-step walkthrough; use this page to layer on the management plane specifics that keep our Resource Manager crates consistent with the rest of the Azure SDK.
+
+## When to reach for this guide
+
+You are in the right place if all of the following are true:
+
+-   Your TypeSpec project targets Resource Manager APIs (`import "@azure-tools/typespec-azure-resource-manager";`).
+-   You plan to ship the resulting crate under the `azure_mgmt_*` naming pattern in this repository.
```

Comment:
We previously agreed on the name `azure_resourcemanager_*`. It was already codified in the [guidelines](https://azure.github.io/azure-sdk/rust_introduction.html#rust-packaging-prefix), which you should also link here.